### ISSUE TITLE
A11y: Items in product listings - Hierarchy missing

### ISSUE THREAD

**Issue by tieupmedia** [shopware/platform#5237]
*PHP Version:* 8.2   | *Shopware Version:* 6.6.6.7  | *Affected area / extension:* Platform(Default) 

 ----

*_Expected behaviour:_*

Listing pages have a proper hierarchy. Each item has a headline tag.

*_Actual behaviour:_*

1. Listing items contain links, headline, image, add to basket buttons etc., that are grouped together as cards, which isn't represented in the document hierarchy.
2. Furthermore, the product names are visually headlines, but in the html no `h`-tags are set. ![Image](https://github.com/user-attachments/assets/4e412239-e88d-4288-a45d-3af197deeb77)

*_How to reproduce:_*

1. Open a product listing.
2. Inspect the source code by markup review or analyze hierarchy with a helper tool like the browser plugin "HeadingsMap".

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-39212**



[created from NEXT-39212, comment 506031]

**Comment by shopwareBot** [comment]
[public] 

 > Status Change: **Open**

 This issue is ready for implementation.

 

[created from NEXT-39212, comment 510211]

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **Low**

 Thank you for your report. After careful consideration, we've concluded that this issue does not require immediate attention or isn't something we can address right now. We prioritize openness and clear communication with our community, so we want to be transparent. We may not resolve all reported issues, and this one might remain unresolved. To stay effective, we periodically clean our backlog, and this issue could be closed in the future.

 For now, we'll keep this ticket open to invite further feedback from the community in case we've underestimated its impact and to allow us to revisit it later.



[created from NEXT-39212, comment 526954]

**Comment by lasomethingsomething** [comment]
Marked resolved in Jira with 6.7.0.0 fix version, so closing
